contracts/exchange/api/TransferContract.sol

```diff
@@ -318,6 +318,7 @@ abstract contract TransferContract is TradeContract {
     _requireSubAccountUnderAccount(toSub, toAccID);
 
     _fundAndSettle(fromSub);
+    _fundAndSettle(toSub);
```

Comment:
Found through assertion that the risk engine F&S any subaccount involved in the transfer, thus adding here.

Reply:
Good guy assertion